CHANGELOG.md

```diff
@@ -25,6 +27,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Deprecated`
 
+- [#670](https://github.com/nf-core/mag/pull/670) - Deprecated --gtdbtk_pplacer_scratch due to uninuitive usage (reported by harper357, fixed by @jfy133)
```

Comment:
Forgot the `@`

Reply:
Done in https://github.com/nf-core/mag/pull/671/commits/459f41853d4fe7e18d9c3eb9500e557e55f79f1d